tools/gce/create_windows_debug_worker.sh

```diff
@@ -49,7 +49,8 @@ gcloud compute instances create "$INSTANCE_NAME" \
     --zone "$ZONE" \
     --machine-type "$MACHINE_TYPE" \
     --image-project google.com:kokoro \
-    --image kokoro-win7build-v9-prod-debug \
+    # The version might need updating.
```

Comment:
I don't think this works.  This line terminates the command above, and the next command is "--image kokoro-win7build........"

Reply:
Yeah.. Good catch!